diff --git a/node_modules/three-to-ammo/index.js b/node_modules/three-to-ammo/index.js
index 62df76e..6ae1b31 100644
--- a/node_modules/three-to-ammo/index.js
+++ b/node_modules/three-to-ammo/index.js
@@ -269,7 +269,7 @@ exports.createHACDShapes = (function() {
       return [];
     }
 
-    const bounds = _computeBounds(root);
+    const bounds = _computeBounds(root, options);
     const scale = _computeScale(root, options);
 
     let vertexCount = 0;
@@ -662,7 +662,7 @@ const _iterateGeometries = (function() {
   const inverse = new THREE.Matrix4();
   const bufferGeometry = new THREE.BufferGeometry();
   return function(root, options, cb) {
-    inverse.getInverse(root.matrixWorld);
+    inverse.copy(root.matrixWorld).invert();
     root.traverse(mesh => {
       if (
         mesh.isMesh &&
